src/group_theory/subgroup/basic.lean

```diff
@@ -2670,6 +2670,16 @@ begin
   exact subset_normal_closure (set.mem_singleton _),
 end
 
+variables (G) {M : Type*} [monoid M]
+
+lemma eq_of_mem_center_left {g h : M} (H : is_conj g h) (Hg : g ∈ set.center M) :
```

Comment:
@YaelDillies, thoughts on `eq_of_left_mem_center` instead?

Reply:
I think it's marginally better, yeah.